recipes/statsforecast/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "statsforecast" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/statsforecast-{{ version }}.tar.gz
+  sha256: 988f6586408127e30930f59a52d4abce2d7794dd03e333b06e8314d6493dfbfd
+
+build:
+  number: 0
+  noarch: python
+  script:
+    ## NOTE: remove the following once the 
+    #        following PR is merged.
+    #  PR: https://github.com/Nixtla/statsforecast/pull/24
+    #
+    ## Ensure inclusion of settings.ini
+    {% set settingsfile = "settings.ini" %}
+    # IF {{ settingsfile }} is NOT PRESENT, COPY {{ settingsfile }} from $RECIPE_DIR
+    - echo -e "\n>>> ENSURE INCLUSION OF '{{ settingsfile }}' <<<\n\n"
+    - if [ ! -f {{ settingsfile }} ]; then cp $RECIPE_DIR/{{ settingsfile }} {{ settingsfile }}; fi  # [not win]
+    - copy {{ environ.get("RECIPE_DIR") }}\{{ settingsfile }} {{ settingsfile }}  # [win]
+    - {{ PYTHON }} -m pip install . -vv
```

```suggestion
source:
  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/statsforecast-{{ version }}.tar.gz
    sha256: 988f6586408127e30930f59a52d4abce2d7794dd03e333b06e8314d6493dfbfd 
    ## NOTE: remove the following once the 
    #        following PR is merged.
    #  PR: https://github.com/Nixtla/statsforecast/pull/24
  - path: settings.ini

build:
  number: 0
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
```

🙏🏼 for explaining the purpose of the script, but also conda has a build in capability of adding files from inside the recipe. Something like above should work. Use this capability in the future. Gonna ignore it this time because you will be removing it before the next release.

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#source-from-a-local-path